doc/source/user_guide/pystan2_refitting_xr_lik.ipynb

```diff
@@ -3181,9 +3181,13 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "We will create a subclass of {class}`~arviz.SamplingWrapper`. Therefore, instead of having to implement all functions required by {func}`~arviz.reloo` we only have to implement `sel_observations` (we are cloning `sample` and `get_inference_data` from the `PyStan2SamplingWrapper` in order to use `apply_ufunc` instead of assuming the log likelihood is calculated within Stan). \n",
+    "We will create a subclass of {class}`~arviz.SamplingWrapper`. Therefore, instead of having to implement all functions required by {func}`~arviz.reloo` we only have to implement {func}`~arviz.SamplingWrapper.sel_observations` (we are cloning  {func}`~arviz.PyStanSamplingWrapper.sample` and {func}`~arviz.PyStanSamplingWrapper.get_inference_data` from the {class}`~arviz.PyStan2SamplingWrapper` in order to use `apply_ufunc` instead of assuming the log likelihood is calculated within Stan). \n",
```

Comment:
we are subclassing the base `SamplingWrapper` here, not the pystan one, so links should be to the base class instead of to pystan one